repos/re-theme/src/hooks/useStylesMemo.js

```diff
@@ -0,0 +1,28 @@
+/** @module hooks */
+
+import { useMemo } from 'react'
+import { useTheme } from './useTheme'
+import { deepFreeze } from '@keg-hub/jsutils'
+
+const noOpObj = deepFreeze({ content: {} })
```

Comment:
* Update this to pull from `../helpers/noOp.js`
* Then in `../helpers/noOp.js` import the `noOpObj` from jsutils

Reply:
updated